airflow-postgres-app-services/azuredeploy.json

```diff
@@ -0,0 +1,263 @@
+            "metadata": {
+                "description": "Tier for Service Plan"
+            }
+        },
+        "postgreDatabaseLogin": {
+            "type": "string",
+            "minLength": 1,
+            "metadata": {
+                "description": "Database administrator login name"
+            }
+        },
+        "postgrePassword": {
+            "type": "securestring",
+            "minLength": 8,
+            "maxLength": 128,
+            "metadata": {
+                "description": "Database administrator password"
+            }
+        },
+        "postgreSkuCapacity": {
+            "type": "int",
+            "defaultValue": 1,
+            "allowedValues": [
+                1,
+                2,
+                4,
+                8,
+                16,
+                32
+            ],
+            "metadata": {
+                "description": "Azure database for PostgreSQL compute capacity in vCores (1, 2, 4, 8, 16, 32)"
+            }
+        },
+        "postgreSkuName": {
+            "type": "string",
+            "defaultValue": "B_Gen5_1",
+            "allowedValues": [
+                "B_Gen5_1",
+                "B_Gen5_2",
+                "GP_Gen5_2",
+                "GP_Gen5_4",
+                "GP_Gen5_8"
+            ],
+            "metadata": {
+                "description": "Azure database for PostgreSQL SKU name "
+            }
+        },
+        "postgreSkuSizeMB": {
+            "type": "int",
+            "allowedValues": [
+                102400,
+                51200,
+                10240,
+                5120
+            ],
+            "defaultValue": 5120,
+            "metadata": {
+                "description": "Azure database for PostgreSQL Sku Size "
+            }
+        },
+        "postgreSkuTier": {
+            "type": "string",
+            "defaultValue": "Basic",
+            "allowedValues": [
+                "Basic",
+                "GeneralPurpose",
+                "MemoryOptimized"
+            ],
+            "metadata": {
+                "description": "Azure database for PostgreSQL pricing tier"
+            }
+        },
+        "postgreVersion": {
+            "type": "string",
+            "allowedValues": [
+                "9.5",
+                "9.6",
+                "10",
+                "11"
+            ],
+            "defaultValue": "11",
+            "metadata": {
+                "description": "PostgreSQL version"
+            }
+        },
+        "postgreSkuFamily": {
+            "type": "string",
+            "defaultValue": "Gen5",
+            "metadata": {
+                "description": "Azure database for PostgreSQL SKU family"
+            }
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "Location for all resources."
+            }
+        },
+        "guidValue": {
```

Comment:
remove this and use an idempotent seed every where you use this (e.g. resourceGroup().id)

Reply:
Updated!